docs/network/interfaces_and_networks.md

```diff
@@ -91,61 +99,61 @@ definition.
 apiVersion: "k8s.cni.cncf.io/v1"
 kind: NetworkAttachmentDefinition
 metadata:
-  name: bridge-test
+  name: linux-bridge-net-ipam
 spec:
   config: '{
       "cniVersion": "0.3.1",
-      "name": "bridge-test",
-      "type": "bridge",
-      "bridge": "br1",
-      "disableContainerInterface": true
+      "name": "mynet",
+      "plugins": [
+        {
+          "type": "bridge",
+          "bridge": "br10",
+          "disableContainerInterface": true,
+          "ipam": {
+            "type": "whereabouts",
+            "range": "10.1.1.0/24"
+          },
+          "macspoofchk": true
+        }
+      ]
     }'
 ```
 
 With following definition, the VMI will be connected to the default pod
-network and to the secondary Open vSwitch network.
+network and to the secondary bridge network, referencing the `NetworkAttachmentDefinition` 
+shown above(in the same namespace)
 
 ```yaml
-kind: VM
+# partial example - kept short for brevity 
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
 spec:
-  domain:
-    devices:
-      interfaces:
-        - name: default
-          masquerade: {}
-          bootOrder: 1   # attempt to boot from an external tftp server
-          dhcpOptions:
-            bootFileName: default_image.bin
-            tftpServerName: tftp.example.com
-        - name: ovs-net
-          bridge: {}
-          bootOrder: 2   # if first attempt failed, try to PXE-boot from this L2 networks
-  networks:
-  - name: default
-    pod: {} # Stock pod network
-  - name: ovs-net
-    multus: # Secondary multus network
-      networkName: ovs-vlan-100
+  template:
+    spec:
+       domain:
+         devices:
+           interfaces:
+           - name: default
+             masquerade: {}
+           - name: bridge-net
+             bridge: {}
+       networks:
+       - name: default
+         pod: {} # Stock pod network
+       - name: bridge-net
+         multus: # Secondary multus network
+           networkName: linux-bridge-net-ipam #ref to NAD name
 ```
-
+#### Multus as primary network provider
 It is also possible to define a multus network as the default pod
-network with [Multus](https://github.com/intel/multus-cni). A version of
-multus after this [Pull
-Request](https://github.com/intel/multus-cni/pull/174) is required
-(currently master).
+network by indicating `spec.networks.multus.default=true`.
```

Comment:
Since the example was converted to a `VirtualMachine` object:
```suggestion
network by indicating `spec.template.spec.networks.multus.default=true`.
```

Reply:
done